### PR TITLE
made tests pass again

### PR DESCRIPTION
**Pull request by quinnabrvau**

### CODE REVIEW

File: test/test.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def setup_production_environment(database, url):
+    tables = [table.name for table in database.resource.tables.all()]
+
+    if 'player-alias-table' not in tables:
+        database.create_support_table('player-alias-table', 'Alias')
+
+    if 'channel-table' not in tables:
+        database.create_support_table('channel-table', 'ChannelName')
+
+    if 'daily-rating-record-table' not in tables:
```
Comment (JimLiu0) [inline test/test.py:19]: Does this not need additional arguments to pass in keys?

Comment (quinnabrvau) [reply test/test.py:19]: The if statement? No, it's just looking to see if a table of that name is in the tables.

The second argument to `create_support_table` is the key which is `ChannelName`.

Comment (quinnabrvau) [reply test/test.py:19]:
```
def create_support_table(self, table_name, key):
        return self.resource.create_table(
            TableName=table_name,
            KeySchema=[
                {'AttributeName': key, 'KeyType': 'HASH'},
            ],
            AttributeDefinitions=[
                {'AttributeName': key, 'AttributeType': 'S'},
            ],
            ProvisionedThroughput={
                'ReadCapacityUnits': 1,
                'WriteCapacityUnits': 1,
            },
            BillingMode='PROVISIONED',
        )
```

Comment (quinnabrvau) [reply test/test.py:19]: oh and `self == database` which is the 1st argument

Comment (JimLiu0) [reply test/test.py:19]: For daily and yesterday tables: `database.create_table('daily-rating-record-table')`. How does the newly created table know what the keys are?

Comment (quinnabrvau) [reply test/test.py:19]: `create_tables` assumes that we are using `PlayerName` and `Region` keys. I have been using this to create all the tables used in the tests, which have a different name then the actual production tables.

```
 def create_table(self, table_name=None):
        if table_name is None:
            table_name = self.table_name
        # print("creating table")
        self.table = self.resource.create_table(
            TableName=table_name,
            KeySchema=[
                {'AttributeName': 'PlayerName', 'KeyType': 'HASH'},
                {'AttributeName': 'Region', 'KeyType': 'RANGE'},
            ],
            AttributeDefinitions=[
                {'AttributeName': 'PlayerName', 'AttributeType': 'S'},
                {'AttributeName': 'Region', 'AttributeType': 'S'},
            ],
            ProvisionedThroughput={
                'ReadCapacityUnits': 10,
                'WriteCapacityUnits': 10,
            },
            BillingMode='PROVISIONED',
        )
```

Comment (JimLiu0) [reply test/test.py:19]: Ah ok, I thought this was a boto function instead of a custom one.

**Pull request merged**